esbuild.mjs

```diff
@@ -0,0 +1,34 @@
+// SOURCE: https://github.com/peterpeterparker/create-ic/blob/main/esbuild.mjs
+import esbuild from 'esbuild';
+import {existsSync, mkdirSync, writeFileSync} from 'fs';
+import {join} from 'path';
+import NodeResolve from '@esbuild-plugins/node-resolve';
+
+const dist = join(process.cwd(), 'dist');
+
+if (!existsSync(dist)) {
+  mkdirSync(dist);
+}
+
+const script = await esbuild.build({
+  entryPoints: ['src/index.ts'],
+  bundle: true,
+  minify: false,
```

Comment:
now that it works out you can `minify: true` or you rather like it not minified?

Reply:
Yes, true. Thanks